tests/CallbackTest.php

```diff
@@ -89,8 +89,8 @@ public function testViewUrlCallback(): void
         $this->simulateCallbackTriggering($cbApp);
         $this->simulateCallbackTriggering($cb);
 
-        $expectedUrlCbApp = '?' . Callback::URL_QUERY_TRIGGER_PREFIX . 'aa=callback&' . Callback::URL_QUERY_TARGET . '=aa';
-        $expectedUrlCb = '?' . /* Callback::URL_QUERY_TRIGGER_PREFIX . 'aa=1&' . */ Callback::URL_QUERY_TRIGGER_PREFIX . 'bb=callback&' . Callback::URL_QUERY_TARGET . '=bb';
+        $expectedUrlCbApp = '/index.php?' . Callback::URL_QUERY_TRIGGER_PREFIX . 'aa=callback&' . Callback::URL_QUERY_TARGET . '=aa';
```

Comment:
This change is not correct as the $_SERVER vars are not correctly set by `simulateCallbackTriggering` method.

Path on line https://github.com/atk4/ui/blob/f1277b7926/src/App.php#L673 is `''` instead of `'/'`, but in properly configured `$_SERVER`, the path must always start with `/`.

Reply:
TODO added in https://github.com/atk4/ui/commit/b3852843ae5387064ac03a3cf70a761ad70ece64#diff-ad46c7350003ccf7f086dccbecc6fcb7706a1558b14eb5e437a856b8c619e71fR673